lemur/tests/helper_random.py

```diff
@@ -0,0 +1,233 @@
+# This file consists of the functions that generate random values or
+# create objects in db for testing
+import sys
+sys.path.append('../..')
+# Libraries
+# Standard library
+from random import choice, randint, shuffle
+from string import ascii_lowercase, ascii_uppercase, digits
+
+# Local
+from lemur import models as m
+from lemur.utility.utility_generate_and_convert import (generate_lab_id,
```

Comment:
I will fix them once I settled down the path of the files switching to another testing framework.